rts/at/sra/README.md

```diff
@@ -0,0 +1,68 @@
+OVERVIEW
+--------
+An example of how to use REST-AT service.
+
+USAGE
+--------------------
+TEST SRA PARTICIPANTS
+
+Start a REST-AT coordinator on port 8080
```

Comment:
What makes it start on port 8080

Reply:
This is updated, so resolving the conversasion.